test/test_container_application.py

```diff
@@ -0,0 +1,194 @@
+        assert cip
+        # This function tests PHP session
+        assert self.s2i_app.test_response(
+            url=cip, page="/session_test.php",
+            expected_output="Passed"
+        )
+        # Assert tests if connect to http://<ip>:8080 works
+        assert self.s2i_app.test_response(
+            url=cip
+        )
+        # Assert tests if connect to https://<ip>:8443 works
+        assert self.s2i_app.test_response(
+            url=f"https://{cip}", port=8443
+        )
+        # test if configuration is writeable for php.ini
+        assert PodmanCLIWrapper.podman_run_command_and_remove(
+            cid_file_name=VARS.IMAGE_NAME,
+            cmd="[ -w \\${PHP_SYSCONF_PATH}/php.ini ]",
+            return_output=False
+        ) == 0
+        # test if php.d directory is writeable
+        assert PodmanCLIWrapper.podman_run_command_and_remove(
+            cid_file_name=VARS.IMAGE_NAME,
+            cmd="[ -w \\${PHP_SYSCONF_PATH}/php.d ]",
+            return_output=False
+        ) == 0
+
+
+class TestPHPClearEnvContainer:
+    def setup_method(self):
+        self.s2i_app = build_s2i_app(test_app, container_args="-e PHP_CLEAR_ENV=OFF")
+
+    def teardown_method(self):
+        if self.s2i_app:
+            self.s2i_app.cleanup()
+
+    def test_run_app_test(self):
+        """
+        Test checks if php-version is proper one
+        and specific pages returns expected message or return code.
+        """
+        skip_clear_env_tests()
+        cid_file_name = self.s2i_app.app_name
+        assert self.s2i_app.create_container(cid_file_name=cid_file_name, container_args="--user=100001")
+        assert ContainerImage.wait_for_cid(cid_file_name=cid_file_name)
+        cid = self.s2i_app.get_cid(cid_file_name=cid_file_name)
+        assert cid
+        assert VARS.VERSION in PodmanCLIWrapper.podman_exec_shell_command(
+            cid_file_name=cid,
+            cmd="php --version"
+        )
+        assert VARS.VERSION in PodmanCLIWrapper.podman_exec_shell_command(
+            cid_file_name=cid,
+            cmd="php --version",
+            used_shell="/bin/sh"
+        )
+
+        cip = self.s2i_app.get_cip(cid_file_name=cid_file_name)
+        assert cip
+        # This function tests PHP session
+        assert self.s2i_app.test_response(
+            url=cip, page="/session_test.php",
+            expected_output="Passed"
+        )
+        # Assert tests if connect to http://<ip>:8080 works
+        assert self.s2i_app.test_response(
+            url=cip
+        )
+        # Assert tests if connect to https://<ip>:8443 works
+        assert self.s2i_app.test_response(
+            url=f"https://{cip}", port=8443
+        )
+        # test if configuration is writeable for php.ini
+        assert PodmanCLIWrapper.podman_run_command_and_remove(
+            cid_file_name=VARS.IMAGE_NAME,
+            cmd="[ -w \\${PHP_SYSCONF_PATH}/php.ini ]",
```

Comment:
I find this more readable
```suggestion
            cmd="test -w \\${PHP_SYSCONF_PATH}/php.ini",
```

Reply:
Have you decided not to implement this change for any particular reason?